MushroomObserver/mushroom-observer
# Normalize GlossaryController

Update `GlossaryController` to Rails 5 standards (as `GlossaryTermsController`)
Includes:
- Standardize names and locations of controllers
- Conform names/locations of tests
- Conform associated view locations and names
- Refactor controllers, routes, tests:
  - Standardize action names
  - Standardize routes
  - When appropriate, divide or namespace controllers

For more details, see https://www.pivotaltracker.com/epic/show/4622322, https://www.pivotaltracker.com/story/show/174310150.

Suggested manual tests
- Click "Glossary" (left navbar) => It should display index
- Click a term => it should display the term
- **Display past version(s)**. (It's important to check this manually because routing (including path- and url helpers) is _sui generis_
- Back to Glossary, Click "Create a Glossary Term", complete the form, Save. => It should create the new term and display it.
- Edit it. 
- Turn on Admin mode. => You should see a Destroy link (a new feature).
- Destroy it.

Delivers https://www.pivotaltracker.com/story/show/174310150
